cpp/dolfinx/fem/interpolate.h

```diff
@@ -1097,14 +1113,16 @@ create_nonmatching_meshes_interpolation_data(const mesh::Mesh<T>& mesh0,
 /// @brief Interpolate from one finite element Function to another one.
 /// @param[out] u The function to interpolate into
 /// @param[in] v The function to be interpolated
-/// @param[in] cells List of cell indices to interpolate on
+/// @param[in] cells_u List of cell indices to interpolate on
+/// @param[in] cell_map Mapping from cells in `u` to cells in `v`
 /// @param[in] nmm_interpolation_data Auxiliary data to interpolate on
 /// nonmatching meshes. This data can be generated with
 /// create_nonmatching_meshes_interpolation_data (optional).
 template <dolfinx::scalar T, std::floating_point U>
 void interpolate(
     Function<T, U>& u, const Function<T, U>& v,
-    std::span<const std::int32_t> cells,
+    std::span<const std::int32_t> cells_u,
```

Comment:
Better names?

Reply:
Do you have a suggestion? We need to make it clear that the cells sent in are associated with the mesh of `u`, while the map maps each cell in `u` to a cell in `v`.

Reply:
Use `u0` and `u1` for the functions, the `cells0` etc.